seb_openedx/templates/seb-manage-users/table.html

```diff
@@ -0,0 +1,31 @@
+{% load django_tables2 %}
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <style>
+      a {
+        color: cadetblue;
+      }
+      .btn-info {
+        background-color: cadetblue;
+        border-color: cadetblue;
+      }
+    </style>
+    <title>Django People</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row justify-content-center">
+        <div class="col-10">
+          <br>
+          <h1 class="mt-2">SEB - User Admin</h1>
+          <br>
+          {% render_table table %}
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Emtpy line

Reply:
Ok

Reply:
still missing the emtpy line

Reply:
I see it again. @github, this is confusing